DeepSpeech.py

```diff
@@ -191,6 +180,7 @@
 n_context = 9 # TODO: Determine the optimal value using a validation data set
 
 # Number of units in hidden layers
+n_hidden = 2048
```

Comment:
Yeah - but this changed now to a new `initialize_globals()` approach. Will soon push an updtate to re-review this.